test/smoke_test/smoke_test.sh

```diff
@@ -3,10 +3,15 @@
 # This script is intended to be run from within the Docker "smoke_test" container
 #
 set -e
-
-go run bcda_client.go decrypt_util.go -host=api:3000 -endpoint=ExplanationOfBenefit
-go run bcda_client.go decrypt_util.go -host=api:3000 -endpoint=Patient
-go run bcda_client.go decrypt_util.go -host=api:3000 -endpoint=Coverage
-go run bcda_client.go decrypt_util.go -host=api:3000 -endpoint=ExplanationOfBenefit -encrypt=false
-go run bcda_client.go decrypt_util.go -host=api:3000 -endpoint=Patient -encrypt=false
-go run bcda_client.go decrypt_util.go -host=api:3000 -endpoint=Coverage -encrypt=false
\ No newline at end of file
+echo $PATH
+echo "Running Coverage Encrypted"
+go run bcda_client.go -host=api:3000 -endpoint=Coverage
+echo "Running EOB Encrypted"
+go run bcda_client.go -host=api:3000 -endpoint=ExplanationOfBenefit
+echo "Running Patient Encrypted"
+go run bcda_client.go -host=api:3000 -endpoint=Patient
+echo "Running Coverage Encrypted"
+go run bcda_client.go -host=api:3000 -endpoint=Coverage
+go run bcda_client.go -host=api:3000 -endpoint=ExplanationOfBenefit -encrypt=false
```

Comment:
Any interest in echoing the unencrypted tests, similar to what you did above?

Reply:
added.